Classes/Tools.php

```diff
@@ -33,7 +33,7 @@
  *
  * @license http://www.gnu.org/licenses/gpl.html GNU General Public License, version 3 or later
  */
-abstract class Tools
+class Tools
```

Comment:
I'd rather keep this class abstract and move the update code in a dedicated update class. This could be a nice opportunity to get rid of old style `class.ext_update.php` and use the SignalSlot approach instead to instantiate a dedicated update class. I noticed users regularly forget to click on the update button anyway...